plugins/modules/digital_ocean_floating_ip.py

```diff
@@ -287,6 +287,19 @@ def create_floating_ips(module, rest):
     if module.params['droplet_id'] is not None:
         payload["droplet_id"] = module.params['droplet_id']
 
+    # Get existing floating IPs
+    response = rest.get('floating_ips/')
+    status_code = response.status_code
+    json_data = response.json
+
+    # Exit unchanged if any of them are assigned to this Droplet already
+    if status_code == 200:
+        for floating_ip in json_data.get('floating_ips', []):
+            if floating_ip.get('droplet', None) is not None:
```

Comment:
Consider making `droplet` a variable here so you don't have to re-index the dictionary.

Reply:
Sounds good, this should be fixed now!